### ISSUE TITLE
Data provided question

### ISSUE THREAD

**Issue by WinnerForDays** [compneurobilbao/bha2#6]
Hello, 
I am a student trying to use your repository for a project.
Using your data from the readme on Zendono.
I am trying to reproduce your results from this paper and trying to run your jupyter notebook files.
However, I am having a lot of problems.

Is this data given, ready for use to run the jupyter notebook files, or do i need to process them using the other code you provided in different repositories?

All I have is this repo and the data file from Zendono.

Thank you for your help.

**Comment by ajimenezmarin** [comment]
Hi!

It's very nice that you are interested in our project :)

The data uploaded in zenodo should be working with the code. What errors are you having? Could you provide us a screenshot with them?

Thanks!

**Comment by WinnerForDays** [comment]
When trying to load the last part of the first notebook, "Finally, we generate the optimal parcellation (oPA) image, and save it in `bha2/data/processed/CONN_SIZE`". This is my error.
![bha2issue](https://github.com/compneurobilbao/bha2/assets/60327206/b077e445-8638-4374-b6e5-cee01c45a4d1)

**Comment by ajimenezmarin** [comment]
Hi!

The repo was thought for starting building the multiscalar trees, so in `Usage` section in the `Readme.md` was described how to build them as initial step. But indeed, It would not be mandatory if a user like you use directly the data from Zenodo.

I have added the code `fix_ipa_rois.py` to be used as an initial step if a user doesn't want to build the trees. I have added also how to use it in `Usage` section.

Thanks for the concern.
